static/app/views/issueDetails/streamline/activitySection.tsx

```diff
@@ -88,9 +86,56 @@ function StreamlinedActivitySection({group}: {group: Group}) {
     [group.activity, mutators, group.id]
   );
 
+  const createTimelineItem = useCallback(
```

Comment:
if possible, pull this out into its own component TimelineItem having it as a sub function kinda breaks how keys work